AEPTarget/Sources/Target.swift

```diff
@@ -428,6 +436,81 @@ public class Target: NSObject, Extension {
         }
     }
 
+    private func rawClickedLocation(_ event: Event) {
+        if isInPreviewMode {
```

Comment:
nit: `is` feels redundant in this name

Reply:
This is existing code, however I think I can modify it without any major impact.